alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertMover.kt

```diff
@@ -111,9 +110,8 @@ class AlertMover(
 
     private fun onCopyResponse(response: BulkResponse) {
         val deleteRequests = response.items.filterNot { it.isFailed }.map {
-            DeleteRequest(AlertIndices.ALERT_INDEX, AlertIndices.MAPPING_TYPE, it.id)
+            DeleteRequest(AlertIndices.ALERT_INDEX, it.id)
                     .routing(monitorId)
-                    .version(it.version)
```

Comment:
Why isn't this `version` replaced with `seqNo` + `primaryTerm`?

Reply:
because `BulkItemResponse` only has `seqNo` and `primaryTerm` when its a `Failure()` which we are filtering out here.

[Reference](https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/action/bulk/BulkItemResponse.java#L178)